### PR TITLE
document cpmm-1

### PR DESCRIPTION
**Pull request by Gurkenglas**

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -61,7 +61,9 @@ What's more, when the creator is free to use their judgment, many new kinds of p
 
 ### **What kind of betting system do you use?**
 
-Manifold Markets uses a special type of automated market marker based on a dynamic pari-mutuel (DPM) betting system.
+Manifold markets is currently using more than one kind of automated market maker.
+
+Markets created before March 15 and Free response markets use a special type of automated market marker based on a dynamic pari-mutuel (DPM) betting system.
```
Comment (akrolsmir) [inline docs/about.md:66]: I'd reword this:

"Free response markets, as well as legacy binary markets (created before Mar 15, 2022), use a dynamic..."

Reasons:
- Focus on the most important thing first (FR markets use DPM); less awkward wording
- Cut "special"; it's just one of two different kinds.

File: docs/about.md [hunk 2]
```diff
@@ -61,7 +61,9 @@ What's more, when the creator is free to use their judgment, many new kinds of p
 
 ### **What kind of betting system do you use?**
 
-Manifold Markets uses a special type of automated market marker based on a dynamic pari-mutuel (DPM) betting system.
+Manifold markets is currently using more than one kind of automated market maker.
```
Comment (akrolsmir) [inline docs/about.md:64]: I'd directly emphasize "Manifold Markets is currently using two kinds of automated market makers".

- Directly specify how many kinds
- Note the capitalization of Markets
- Bonus: Could link to a writeup of AMMs eg https://www.coindesk.com/learn/2021/08/20/what-is-an-automated-market-maker/

File: docs/about.md [hunk 3]
```diff
@@ -71,6 +73,8 @@ The result is a market that can function well when trading volume is low without
 
 Read **[our technical guide](https://www.notion.so/Technical-Guide-to-Manifold-Markets-b9b48a09ea1f45b88d991231171730c5)** to find out more!
 
+Binary markets created after March 15 use a variation on constant-product market makers due to Manifold user Pepe: Instead of the product (or equivalently, the geometric mean) of YES shares and NO shares on the table, they hold constant the weighted geometric mean, with weights equal to the probabilities chosen by the market creator at creation.
```
Comment (akrolsmir) [inline docs/about.md:76]: Would reword as:

"Newer binary markets use a constant function market maker, by the formula .... This hold constant the weighted geometric mean, with weights equal to the probabilities chosen by the market creator at creation.

This design was inspired by Uniswap's CPMM and a suggestion from Manifold user Pepe" (and then link to the appropriate places).

File: docs/about.md (hunk 2 above)
Comment (akrolsmir) [reply docs/about.md:64]: One more suggestion: It'd be good to add further structure in the form of numbers here.

Manifold Markets is currently using two kinds of automated market makers.

1. **Dynamic Parimutuel (DPM)**
    Free response markets, ...
 
2. **Constant Function Market Maker (CFMM)**
    ...

**Review by akrolsmir — CHANGES_REQUESTED** [review]
Thanks a lot for clarifying this! Our documentation here is woefully out of date here.

Also flagging @mantikoros as this is his area of expertise. (And as a reminder to update the technical guide when he has time)

File: docs/about.md (hunk 3 above)
Comment (akrolsmir) [reply docs/about.md:76]: Reasons:
- Focus first on what these are, because that's what the user cares about
- Assign credit at the end, not as an aide

**Comment by mqp** [comment]
@akrolsmir @mantikoros I want to move these docs into our main repo, so let's figure out what to do with this. Should we merge as-is and work it out later? Should I make the changes you suggested and then merge?

File: docs/about.md (hunk 1 above)
Comment (mantikoros) [reply docs/about.md:66]: All legacy binary markets were converted to Maniswap. Only FR and numeric markets use DPM.

File: docs/about.md (hunk 2 above)
Comment (mantikoros) [reply docs/about.md:64]: 2. Maniswap

**Comment by mqp** [comment]
I agree with the suggestions in these comments. I am going to make a branch based on this PR incorporating these suggestions and then merge.

**Comment by mqp** [comment]
See #7.
